hf-llm-inference/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ghcr.io/coreweave/ml-containers/torch:afecfe9-base-cuda12.0.1-torch2.0.0-vision0.15.1
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update && apt upgrade -y && \
+    apt update && apt install -y python3 python3-pip git curl && \
+    apt clean
+
+RUN mkdir /app
+WORKDIR /app
+
+ARG COMMIT=master
+RUN git clone https://github.com/coreweave/kubernetes-cloud && \
```

Comment:
Cloning the full git history of `kubernetes-cloud` is excessive because of the weird situation with all the GitBook docs assets being thrown in there until recently; you can make it quicker (hundreds fewer megabytes) by using `--filter=blob:none`. It can be even more selective if you copy [the way I wrote the clone in the `llm-inference` branch](https://github.com/coreweave/ml-containers/blob/d1e0d72c435c20b4554175623a78a68980a73284/llm-finetuner/Dockerfile#L11-L22), but `--filter=blob:none` gets most of it.

```suggestion
RUN git clone --filter=blob:none https://github.com/coreweave/kubernetes-cloud && \
```